server/src/main/java/org/elasticsearch/health/HealthIndicatorImpact.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.health;
+
+import org.elasticsearch.xcontent.ToXContentObject;
+
+public interface HealthIndicatorImpact extends ToXContentObject {
```

Comment:
I don't think we need an interface here. Do we envision multiple implementations?

Reply:
The EMPTY implementation was actually a separate one. I've merged them together into a single record.